Gemfile

```diff
@@ -12,5 +12,6 @@ elsif RUBY_VERSION =~ /^1/
   gem 'tins',           '~> 1.6.0'
   gem 'term-ansicolor', '< 1.4'
 end
+gem 'i18n', '1.2.0' if RUBY_VERSION =~ /^2\.[012]/
```

Comment:
`RUBY_VERSION < '2.3'` should do the work and include 1.9

Reply:
Also, will this work outside of TravisCI? As far as I remember `Gemfile` entries were ignored when using a gem

Reply:
The `Gemfile` is used by Bundler, so TravisCI and developers of the Money gem.

Reply:
Looks like the Ruby 1.9.3 job is installing i18n v0.9.5 for other reasons.
https://travis-ci.org/RubyMoney/money/jobs/476150499#L522

Reply:
Change applied in 42f1745.